mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulContestsScreen.kt

```diff
@@ -72,8 +75,18 @@ private fun createContest(
  */
 @Composable
 fun StatefulTournamentScreen(
+    onNewContestClickAction: () -> Unit,
     modifier: Modifier = Modifier,
     contentPadding: PaddingValues = PaddingValues(),
 ) {
-  ContestScreen(TournamentScreenState(), modifier, key = { it.uid }, contentPadding)
+  val state =
+      remember(
+          onNewContestClickAction,
```

Comment:
Missing `rememberUpdatedState`

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/336/commits/92568de2e02f0f445dfba801cf31ae9e98b9a5f4